tests/test_symbolic_verifier.py

```
@@ -570,25 +570,21 @@ def simple(x: int) -> int:
     return x + 1
 """
         result = self.verifier.verify_bounded(code, loop_bound=5, recursion_depth=3)
-        assert "bounded" in result
-        assert result["bounded"] == True
-        assert "bounds_applied" in result
-        assert result["bounds_applied"]["loop_bound"] == 5
-        assert result["bounds_applied"]["recursion_depth"] == 3
-        # All branches share the DiagnosticResult.to_dict() shape.
-        assert "is_verified" in result
-        assert result["status"] in (DiagnosticStatus.UNVERIFIABLE.value, DiagnosticStatus.BLOCKED.value)
+        assert result.developer_fields["bounded"] == True
```

Ruff (0.15.21)

[error] Avoid equality comparisons to `True`; use `result.developer_fields["bounded"]:` for truth checks

Replace with `result.developer_fields["bounded"]`

(E712)